DailyKit/CallManager/CallClientSubjects.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import Combine
+import Daily
+
+/// Subjects for ``CallClientProperty``s.
+final class CallClientSubjects {
+    let url: CurrentValueSubject<URL?, Never>
+    let callState: CurrentValueSubject<CallState, Never>
+    let camera: CurrentValueSubject<CallCamera, Never>
+    let microphone: CurrentValueSubject<CallMicrophone, Never>
+    let participants: CurrentValueSubject<CallParticipants, Never>
+
+    private var publishers: [AnyHashable: Any] = [:]
+
+    init(
+        url: URL?,
+        callState: CallState,
+        camera: CallCamera,
+        microphone: CallMicrophone,
+        participants: CallParticipants
+    ) {
+        self.url = CurrentValueSubject<URL?, Never>(url)
+        self.callState = CurrentValueSubject<CallState, Never>(callState)
+        self.camera = CurrentValueSubject<CallCamera, Never>(camera)
+        self.microphone = CurrentValueSubject<CallMicrophone, Never>(microphone)
+        self.participants = CurrentValueSubject<CallParticipants, Never>(participants)
```

Comment:
You shouldn't need the fully qualified names here, I think?

```suggestion
        self.url = .init(url)
        self.callState = .init(callState)
        self.camera = .init(camera)
        self.microphone = .init(microphone)
        self.participants = .init(participants)
```